src/source/mod.rs

```diff
@@ -241,15 +241,28 @@ where
     ///
     /// # Parameters
```

Comment:
From some research the documentation for docs.rs will be generated of these comments correct?

Reply:
yes rust is great like that :)

You can also open/generate the docs locally (will check some things for you too) using: `cargo doc --open`.

Reply:
Thanks working on making the styling better for the method automatic_gain_control